coq-core.opam

```diff
@@ -34,6 +34,17 @@ depends: [
   "ounit2" {with-test}
 ]
 build: [
+  [
+    "./configure"
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
```

Comment:
If we're passing this stuff we don't need the COQ_CONFIGURE_PREFIX mechanism.

Reply:
Note `-docdir` here used to be bogus, can't remeber what's the right one.